packages/keycodes/src/platform.native.js

```diff
@@ -0,0 +1,13 @@
+import { Platform } from 'react-native';
+
+/**
+ * Return true if platform is MacOS.
+ *
+ * @param {Object} _window   window object by default; used for DI testing.
+ *
+ * @return {boolean}         True if iOS; false otherwise.
+ */
+// eslint-disable-next-line no-unused-vars
+export function isAppleOS( _window = window ) {
```

Comment:
To make it even simpler you can do the following:

FIle name: `platform.ios.js`:
```js
export const isMacOs = () => true;
```
FIle name: `platform.android.js`:
```js
export const isMacOs = () => false;
```